query/stats.go

```diff
@@ -57,53 +57,41 @@ func (s *Stat) reset() *Stat {
 
 // statGroup collects simple streaming statistics.
 type statGroup struct {
-	min    float64
-	max    float64
-	mean   float64
+	latencyHDRHistogram *hdrhistogram.Histogram
 	sum    float64
 	values []float64
```

Comment:
values is kept, but other than the initial/first recorded value, the others are not appended to the array